<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectModeUi</class>
 <widget class="QDialog" name="SelectModeUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create or Import User</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <spacer name="spacer2_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&lt;qt&gt;&lt;p&gt;You can now setup a new online banking user.&lt;/p&gt;
&lt;p&gt;Please note that AqHBCI/ABanking can only import keyfiles which have been created with OpenHBCI- or AqBanking-based applications.&lt;/p&gt;
&lt;p&gt;Keyfiles of proprietary software - like Moneyplex, Starmoney etc - can not be used, since the manufacturers of those programs do not publish their file formats.&lt;/p&gt;
&lt;p&gt;Keyfiles created by Gnucash, KMymoney, AqMoney, KOpenHBCI, GOpenHBCI, LxBank, AqHBCI-Tool and QBankManager can be used.&lt;/p&gt;
&lt;/qt&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Q3ButtonGroup" name="buttonGroup1">
     <property name="title">
      <string>Create or Import User</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1" rowspan="2">
       <widget class="QRadioButton" name="importFileRadio">
        <property name="toolTip">
         <string>Import an existing keyfile</string>
        </property>
        <property name="text">
         <string>Import Keyfile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="createFileRadio">
        <property name="toolTip">
         <string>Create a new keyfile</string>
        </property>
        <property name="text">
         <string>Create Keyfile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="importCardRadio">
        <property name="toolTip">
         <string>Use with DDV cards or personalized RSA cards</string>
        </property>
        <property name="text">
         <string>Import Chipcard</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="pinTanRadio">
        <property name="text">
         <string>Setup PIN/TAN</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QRadioButton" name="initCardRadio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use with completely new RSA cards</string>
        </property>
        <property name="text">
         <string>Init Chipcard</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>importCardRadio</tabstop>
  <tabstop>importFileRadio</tabstop>
  <tabstop>initCardRadio</tabstop>
  <tabstop>createFileRadio</tabstop>
  <tabstop>pinTanRadio</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>SelectModeUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>SelectModeUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
